response.go

```diff
@@ -15,6 +15,7 @@ type Response struct {
 	buf  smallBuf
 }
 
+//nolint: unused
```

Comment:
AFAIS, it does not affect anything.

Reply:
Fixed.

Reply:
Ouch, I see now: it is the real dead code. Let's remove it so?

Reply:
Removed.